src/components/MapGL/index.js

```diff
@@ -502,6 +503,25 @@ class MapGL extends PureComponent<Props, State> {
     }
   }
 
+  /**
+   * Update Map size from newProps
+   *
+   * @private
+   * @param {Props} prevProps
+   * @param {Props} newProps
+   */
+  _updateMapSize(prevProps: Props, newProps: Props): void {
+    const sizeChanged =
+      prevProps.style?.height !== newProps.style?.height ||
+      prevProps.style?.width !== newProps.style?.width;
+
+    if (!sizeChanged) {
+      return;
+    }
+
+    this._map.resize();
```

Comment:
@stepankuzmin just wanted to make sure you saw my note about this. During tests, this call will actually fail despite working fine in a real browser. Any thoughts on why that is and how I could get the test working?